### PR TITLE
 Fix Issue #216, Add New News items to website

### PR DESCRIPTION
**Pull request by IamTregsthedev**
This Pull request fixes issue #216

### CODE REVIEW

**Comment by IamTregsthedev** [comment]
Travis Builds are Failing @kenny2github ?

File: src/views/index/index.jsx [hunk 1]
```diff
@@ -81,19 +81,19 @@ const Index = () => {
...
-                        News
+                       News
                     </div>
                     <div className="content-news-body">
                         <a
-                            href="https://medium.com/scratchfoundation-blog/10-things-to-try-right-now-with-your-child-on-scratch-and-scratchjr-defd7f8ad0d1"
+                            href="https://www.amazon.com/Official-ScratchJr-Book-Help-Learn/dp/1593276710/"
                         >
-                        10 things to try right now with your child on Scratch and ScratchJr
+                      The Official ScratchJr Book is the perfect companion to the app and makes coding easy and fun for all
```
Comment (chrisgarrity) [inline src/views/index/index.jsx]: Close the `</a>` after 'Book'. That is replace the line below with:
`</a> is the perfect companion to the app and makes coding easy and fun for all.`, and remove the duplicate part of the text above.

File: src/views/index/index.jsx [hunk 2]
```diff
@@ -81,19 +81,19 @@ const Index = () => {
...
                         </a>, hands-on activities for getting started with ScratchJr.
                     </div>
                     <div className="content-news-body">
-                        Check out tips for creating with your child: <a href="https://medium.com/scratchfoundation-blog/creating-together-3041662c970a">
-                        Creating Together
-                        </a> by the Scratch Foundation.
+                       <a href="https://www.amazon.com/ScratchJr-Coding-Cards-Creative-Activities/dp/1593278993/"> The ScratchJr Coding Cards are a deck of 75 activity cards covering fun and exciting projects designed to educate young children with ScratchJr.
+
```
Comment (chrisgarrity) [inline src/views/index/index.jsx]: Close the `</a>` link after 'Cards', and have the rest of the text follow the link.

**Comment by chrisgarrity** [comment]
@IamTregsthedev I pushed an update to your branch to fix the formatting errors so that this can be merged. Once I confirm it passes on Travis, I'll merge it. Thanks for your help.

**Pull request merged**